aws/dms/tasks.yml

```
@@ -51,6 +51,7 @@ cron-pii:
 - dashboard.learning_goal_ai_evaluations
 - dashboard.learning_goal_teacher_evaluations
 - dashboard.learning_goal_ai_evaluation_feedbacks
+- dashboard.aitutorinteraction_feedbacks
```

This table is actually called `ai_tutor_interaction_feedbacks`. There are a couple of ways to discover that. The [ActiveRecord model](https://github.com/code-dot-org/code-dot-org/blob/12814a8249fec8c9589c3ed28a47a97c392f0d8d/dashboard/app/models/ai_tutor_interaction_feedback.rb) lists it's table name, for example. You could also check [schema.rb](https://github.com/code-dot-org/code-dot-org/blob/10adb73681521aac29d80e4f3c5e039fafb1d573/dashboard/db/schema.rb). I ran `select tables;` on the database to check. 

```suggestion
- dashboard.ai_tutor_interaction_feedbacks
```